SentimentAnalysis.ipynb

```diff
@@ -21,7 +21,7 @@
    "source": [
     "!pip3 install thirdai --upgrade\n",
     "!pip3 install datasets==2.6.2 \n",
-    "\n",
+    "!pip3 install gradio\n",
```

Comment:
Could we add a markdown cell explaining introducing the gradio demo (in both notebooks)?